### ISSUE TITLE
Translated your application into Russian

### ISSUE THREAD

**Issue by dmaliog** [ubuntu-flutter-community/musicpod#594]
Translated your application into Russian.

Файл: app_ru.arb

```
{
    "@@locale": "ru",
    "play": "Воспроизвести",
    "pause": "Пауза",
    "stop": "Стоп",
    "shuffle": "Перемешать",
    "repeat": "Повторить",
    "repeatAll": "Повторить все",
    "next": "Следующий",
    "back": "Назад",
    "fastForward30": "Перемотать вперёд на 30 секунд",
    "rewind10": "Перемотать назад на 10 секунд",
    "fullWindow": "Перейти в полноэкранный режим",
    "leaveFullWindow": "Выйти из полноэкранного режима",
    "playbackRate": "Скорость воспроизведения",
    "addToFavorites": "Добавить в избранное",
    "removeFromFavorites": "Удалить из избранного",
    "share": "Поделиться",
    "localAudio": "Локальная музыка",
    "localAudioDescription": "Ваша коллекция музыки, хранящаяся на вашем компьютере.",
    "localAudioSubtitle": "Не требуется подключение к интернету",
    "music": "Музыка",
    "radio": "Радио",
    "podcasts": "Подкасты",
    "podcast": "Подкаст",
    "likedSongs": "Понравившиеся песни",
    "likedSongsDescription": "Вся аудиозапись, которая вам понравилась. Локальная или из интернета.",
    "likedSongsSubtitle": "Нажмите кнопку сердца, чтобы добавить больше заголовков.",
    "cancel": "Отмена",
    "add": "Добавить",
    "addTo": "Добавить в:",
    "deletePlaylist": "Удалить плейлист",
    "createNewPlaylist" : "Создать новый плейлист",
    "playlistDialogTitleNew": "Добавить плейлист",
    "playlistDialogTitleEdit": "Редактировать плейлист",
    "save": "Сохранить",
    "title": "Название",
    "titles": "Названия",
    "artist": "Исполнитель",
    "artists": "Исполнители",
    "album": "Альбом",
    "albums": "Альбомы",
    "genres": "Жанры",
    "genre": "Жанр",
    "years": "Годы",
    "year": "Год",
    "albumArtist": "Исполнитель альбома",
    "albumArtists": "Исполнители альбомов",
    "track": "Трек",
    "trackNumber": "Номер трека",
    "diskNumber": "Номер CD",
    "totalDisks": "CD",
    "searchLocalAudioHint": "Поиск локальной музыки",
    "library": "Библиотека",
    "playlists": "Плейлисты",
    "playlist": "Плейлист",
    "discover": "Обнаружить",
    "forYou": "Для вас",
    "search": "Поиск",
    "noPodcastFound": "Извините, подкаст с таким запросом не найден.",
    "noPodcastChartsFound": "Извините, для этой страны или категории нет доступных чартов.",
    "noPodcastSubsFound": "Вы еще не подписались на какие-либо подкасты.",
    "charts": "Чарты",
    "upNext": "Следующий",
    "all": "Все",
    "arts": "Искусство",
    "business": "Бизнес",
    "comedy": "Комедия",
    "education": "Образование",
    "fiction": "Художественная литература",
    "government": "Правительство",
    "healthAndFitness": "Здоровье и фитнес",
    "history": "История",
    "kidsAndFamily": "Дети и семья",
    "leisure": "Досуг",
    "news": "Новости",
    "religionAndSpirituality": "Религия и духовность",
    "science": "Наука",
    "societyAndCulture": "Общество и культура",
    "sports": "Спорт",
    "tvAndFilm": "ТВ и кино",
    "technology": "Технологии",
    "trueCrime": "Криминал",
    "offline": "Офлайн",
    "offlineDescription": "Похоже, ваш компьютер не подключен к интернету.",
    "newEpisodeAvailable": "Доступен новый эпизод:",
    "noStationFound": "Извините, с таким запросом станции не найдены",
    "nothingFound": "Ничего не найдено",
    "noStarredStations": "Вы еще не добавили ни одной станции в избранное.",
    "tags": "Теги",
    "quality": "Качество",
    "station": "Станция",
    "stations": "Станции",
    "country": "Страна",
    "tag": "Тег",
    "failedToImport": "Не удалось импортировать следующие файлы:",
    "unknown": "Неизвестно",
    "volume": "Громкость",
    "queue": "Очередь",
    "limit": "Лимит",
    "decreaseSearchLimit": "Пожалуйста, уменьшите лимит поиска для",
    "podcastFeedIsEmpty": "Извините, лента этого подкаста пуста.",
    "video": "Видео",
    "ok": "OK",
    "noLocalTitlesFound": "Похоже, ваша локальная музыкальная коллекция пуста. Проверьте расположение вашей библиотеки в настройках.",
    "noLocalSearchFound": "Извините, по вашему запросу локальная музыка не найдена.",
    "buyMusicOnline": "Возможно, здесь вы найдете музыку, которую хотели бы купить:",
    "settings": "Настройки",
    "findUsOnGitHub": "Найдите нас на GitHub",
    "musicPodSubTitle": "Ваш музыкальный, радио и подкаст плеер",
    "pickMusicCollection": "Выберите вашу музыкальную коллекцию",
    "newEpisode": "Новый",
    "dontShowAgain": "Не показывать снова",
    "queueConfirmMessage": "Вы действительно хотите добавить {length} медиа в очередь?",
    "@queueConfirmMessage": {
        "placeholders": {
            "length": {
                "type": "String"
            }
        }
    },
    "emptyPlaylist": "Этот плейлист пуст.",
    "copiedToClipBoard": "Скопировано в буфер обмена:",
    "insertIntoQueue": "Добавить в очередь",
    "about": "О программе",
    "localAudioCacheSuggestion": "У вас есть {length} локальных аудиофайлов. Хотите создать кэш для улучшения времени запуска?",
    "@localAudioCacheSuggestion": {
        "placeholders": {
            "length": {
                "type": "String"
            }
        }
    },
    "noThankYou": "Нет, спасибо.",
    "recreateLocalAudioCache": "Создать кэш локальной аудиозаписи заново",
    "useALocalAudioCache": "Использовать локальный кэш аудиозаписи",
    "newEpisodes": "Новые эпизоды",
    "collection": "Коллекция",
    "addToCollection": "Добавить в коллекцию",
    "removeFromCollection": "Удалить из коллекции",
    "loadingPodcastFeed": "Загрузка подкастов...",
    "downloadStarted": "Загрузка началась: {name}",
    "@downloadStarted": {
        "placeholders": {
            "name": {
                "type": "String"
            }
        }
    },
    "downloadCancelled": "Загрузка отменена: {name}",
    "@downloadCancelled": {
        "placeholders": {
            "name": {
                "type": "String"
            }
        }
    },
    "downloadFinished": "Загрузка завершена: {name}",
    "@downloadFinished": {
        "placeholders": {
            "name": {
                "type": "String"
            }
        }
    },
    "downloadsOnly": "Только загрузки",
    "moreOptions": "Дополнительные опции",
    "noRadioServerFound": "Сервер радио не найден",
    "connectedTo": "Подключено к",
    "tryReconnect": "Попробуйте переподключиться",
    "addedTo": "Добавлено в",
    "addToPlaylist": "Добавить в плейлист",
    "open": "Открыть",
    "removeFrom": "Удалить из",
    "noCountryFound": "Страна с таким названием не найдена. Вы также можете использовать телефонные префиксы, такие как +49",
    "noStarredTags": "Вы еще не добавили ни один тег в избранное",
    "name": "Имя",
    "state": "Состояние",
    "playNext": "Воспроизвести следующий",
    "contributors": "Участники",
    "version": "Версия",
    "theme": "Тема",
    "license": "Лицензия",
    "dependencies": "Зависимости",
    "light": "Светлая",
    "system": "Системная",
    "dark": "Тёмная",
    "podcastProvider": "Поставщик подкастов",
    "iTunes": "iTunes",
    "podcastIndex": "Подкаст Индекс",
    "usePodcastIndex": "Использовать Подкаст Индекс вместо iTunes",
    "select": "Выбрать",
    "requiresAppRestart": "Требуется перезапуск приложения",
    "musicCollectionLocation": "Местоположение музыкальной коллекции",
    "astronomyXXXPodcastIndexOnly": "Астрономия",
    "automotiveXXXPodcastIndexOnly": "Автомобильное дело",
    "aviationXXXPodcastIndexOnly": "Авиация",
    "baseballXXXPodcastIndexOnly": "Бейсбол",
    "basketballXXXPodcastIndexOnly": "Баскетбол",
    "beautyXXXPodcastIndexOnly": "Красота",
    "booksXXXPodcastIndexOnly": "Книги",
    "buddhismXXXPodcastIndexOnly": "Буддизм",
    "careersXXXPodcastIndexOnly": "Карьера",
    "chemistryXXXPodcastIndexOnly": "Химия",
    "christianityXXXPodcastIndexOnly": "Христианство",
    "climateXXXPodcastIndexOnly": "Климат",
    "commentaryXXXPodcastIndexOnly": "Комментарий",
    "coursesXXXPodcastIndexOnly": "Курсы",
    "craftsXXXPodcastIndexOnly": "Ремесла",
    "cricketXXXPodcastIndexOnly": "Крикет",
    "cryptocurrencyXXXPodcastIndexOnly": "Криптовалюта",
    "cultureXXXPodcastIndexOnly": "Культура",
    "dailyXXXPodcastIndexOnly": "Ежедневно",
    "designXXXPodcastIndexOnly": "Дизайн",
    "documentaryXXXPodcastIndexOnly": "Документальное кино",
    "dramaXXXPodcastIndexOnly": "Драма",
    "earthXXXPodcastIndexOnly": "Земля",
    "entertainmentXXXPodcastIndexOnly": "Развлечения",
    "entrepreneurshipXXXPodcastIndexOnly": "Предпринимательство",
    "familyXXXPodcastIndexOnly": "Семья",
    "fantasyXXXPodcastIndexOnly": "Фэнтези",
    "fashionXXXPodcastIndexOnly": "Мода",
    "filmXXXPodcastIndexOnly": "Фильм",
    "fitnessXXXPodcastIndexOnly": "Фитнес",
    "foodXXXPodcastIndexOnly": "Еда",
    "footballXXXPodcastIndexOnly": "Футбол",
    "gamesXXXPodcastIndexOnly": "Игры",
    "gardenXXXPodcastIndexOnly": "Садоводство",
    "golfXXXPodcastIndexOnly": "Гольф",
    "healthXXXPodcastIndexOnly": "Здоровье",
    "hinduismXXXPodcastIndexOnly": "Индуизм",
    "hobbiesXXXPodcastIndexOnly": "Хобби",
    "hockeyXXXPodcastIndexOnly": "Хоккей",
    "homeXXXPodcastIndexOnly": "Дом",
    "howToXXXPodcastIndexOnly": "Как сделать",
    "improvXXXPodcastIndexOnly": "Импровизация",
    "interviewsXXXPodcastIndexOnly": "Интервью",
    "investingXXXPodcastIndexOnly": "Инвестиции",
    "islamXXXPodcastIndexOnly": "Ислам",
    "journalsXXXPodcastIndexOnly": "Журналы",
    "judaismXXXPodcastIndexOnly": "Иудаизм",
    "kidsXXXPodcastIndexOnly": "Дети",
    "languageXXXPodcastIndexOnly": "Язык",
    "learningXXXPodcastIndexOnly": "Обучение",
    "lifeXXXPodcastIndexOnly": "Жизнь",
    "managementXXXPodcastIndexOnly": "Управление",
    "mangaXXXPodcastIndexOnly": "Манга",
    "marketingXXXPodcastIndexOnly": "Маркетинг",
    "mathematicsXXXPodcastIndexOnly": "Математика",
    "medicineXXXPodcastIndexOnly": "Медицина",
    "mentalXXXPodcastIndexOnly": "Психика",
    "naturalXXXPodcastIndexOnly": "Натуральный",
    "natureXXXPodcastIndexOnly": "Природа",
    "nonProfitXXXPodcastIndexOnly": "Некоммерческий",
    "nutritionXXXPodcastIndexOnly": "Питание",
    "parentingXXXPodcastIndexOnly": "Воспитание детей",
    "performingXXXPodcastIndexOnly": "Выступления",
    "personalXXXPodcastIndexOnly": "Личное",
    "petsXXXPodcastIndexOnly": "Домашние животные",
    "philosophyXXXPodcastIndexOnly": "Философия",
    "physicsXXXPodcastIndexOnly": "Физика",
    "placesXXXPodcastIndexOnly": "Места",
    "politicsXXXPodcastIndexOnly": "Политика",
    "relationshipsXXXPodcastIndexOnly": "Отношения",
    "religionXXXPodcastIndexOnly": "Религия",
    "reviewsXXXPodcastIndexOnly": "Отзывы",
    "rolePlayingXXXPodcastIndexOnly": "Ролевые игры",
    "rugbyXXXPodcastIndexOnly": "Регби",
    "runningXXXPodcastIndexOnly": "Бег",
    "selfImprovementXXXPodcastIndexOnly": "Самосовершенствование",
    "sexualityXXXPodcastIndexOnly": "Сексуальность",
    "soccerXXXPodcastIndexOnly": "Футбол",
    "socialXXXPodcastIndexOnly": "Социальное",
    "societyXXXPodcastIndexOnly": "Общество",
    "spiritualityXXXPodcastIndexOnly": "Духовность",
    "standUpXXXPodcastIndexOnly": "Стендап",
    "storiesXXXPodcastIndexOnly": "Истории",
    "swimmingXXXPodcastIndexOnly": "Плавание",
    "tVXXXPodcastIndexOnly": "ТВ",
    "tabletopXXXPodcastIndexOnly": "Настольные игры",
    "tennisXXXPodcastIndexOnly": "Теннис",
    "travelXXXPodcastIndexOnly": "Путешествия",
    "videoGamesXXXPodcastIndexOnly": "Видеоигры",
    "visualXXXPodcastIndexOnly": "Визуальное",
    "volleyballXXXPodcastIndexOnly": "Волейбол",
    "weatherXXXPodcastIndexOnly": "Погода",
    "wildernessXXXPodcastIndexOnly": "Дикая природа",
    "wrestlingXXXPodcastIndexOnly": "Борьба",
    "updateAvailable": "Доступно обновление",
    "showMetaData": "Показать метаданные",
    "metadata": "Метаданные",
    "writeMetadata": "Записать метаданные",
    "reorder": "Переупорядочить",
    "move": "Переместить",
    "pinAlbum": "Закрепить альбом в боковой панели",
    "playAll": "Воспроизвести все",
    "hearingHistory": "История прослушивания",
    "emptyHearingHistory": "Вы еще не слушали радио в этой сессии приложения",
    "searchForRadioStationsWithGenreName": "Поиск радиостанций с этим жанром",
    "clearPlaylist": "Удалить все медиа из плейлиста",
    "editPlaylist": "Редактировать плейлист",
    "stationUrl": "URL станции",
    "podcastFeedUrl": "URL ленты подкаста",
    "stationName": "Название станции",
    "podcastName": "Название подкаста",
    "url": "URL",
    "loadFromFileOptional": "Загрузить из файла (необязательно)"
}

```

**Comment by Feichtmeier** [comment]
Thanks @dmaliog !

Appreciate your contribution.

Could you eventually make a pull request so this contribution would be marked with your git user?

**Comment by dmaliog** [comment]
I'm not very good at github to be honest.

**Comment by dmaliog** [comment]
I just tried this.

**Comment by Feichtmeier** [comment]
Yes thank you !
Fixed by #595
